src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -56,6 +57,10 @@ public function __construct($class = null, array $arguments = array())
     {
         $this->class = $class;
         $this->arguments = $arguments;
+        $this->deprecation = array(
```

Comment:
I'm not quite sure I should cluster the property list, as this is really the deprecation configuration, but the change is also fine by me.